tests/settings/manager_test.py

```diff
@@ -136,10 +137,67 @@ def test_dont_choke_on_regex_special_chars_in_tagstring(self):
                         normal = '','', 'white','light red', 'white','#d66'
                 """.format(tag=tag)))
         self.addCleanup(os.unlink, f.name)
+        with self.assertRaises(re.error):
```

Comment:
Which line raises the exception? I suppose the last so you should just wrap the last one with the `with` block.